community/osh/build

```diff
@@ -3,7 +3,7 @@
 :> nostrip
 
 ./configure \
-    --prefix=/usr
+    --prefix /usr
```

Comment:
Intentional?

Reply:
Usage shows `=` https://github.com/oilshell/oil/blob/master/configure#L38

Reply:
I was following the documentation http://www.oilshell.org/release/latest/doc/INSTALL.html I will ask the maintainers of osh.

Reply:
Both work. I think it's best to use `=` anyway. I don't know why both syntaxes are even supported... the usage in this PR is undocumented too.

Reply:
See: https://github.com/oilshell/oil/blob/master/configure#L96-L107